src/style.css

```diff
@@ -0,0 +1,170 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --white-color: #fff;
+  --dark-blue-color: #060b35;
+  --gray-color: #616070;
+  --light-blue-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.container {
+  margin: 0 auto;
+  width: min-content;
+  border-radius: 5px;
+}
+
+.container__card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image-container {
+  margin: 32px 19px 36px;
```

Comment:
More correct add paddings to card instead of adding margins to each element
Smth like this
![image](https://user-images.githubusercontent.com/54936044/224304157-e73cdaa2-06aa-43ca-a785-f975de75ebb8.png)